Automattic/wp-calypso
# Renames DIFM Lite option keys

#### Changes proposed in this Pull Request

This PR will be merged into the parent PR: #59895.

This change reduces duplication.

#### Testing instructions



* Start the flow at `/start/do-it-for-me`.
* Ensure no JS errors.
